pallets/fungibles/src/lib.rs

```diff
@@ -0,0 +1,64 @@
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
Can we start with a single crate named `pallets/api` and work from there? I am concerned about the additional boilerplate that might be required with loads of pallets with a few functions. Finding the balance will come with time and we can decide before progressing beyond devnet.

Also, https://github.com/paritytech/polkadot-sdk/tree/master/polkadot/runtime/parachains/src gives an example how pallets can be defined within a single crate, per module. I think we should ultimately follow this type of approach.

Reply:
See latest commit. 

What do we prefer, like it is now or have a `fungibles` folder with `mod.rs` (like in the pop api crate)?

Reply:
Assuming we will have a separate tests.rs submodule, probably fungibles/mod.rs. My subjective preference is to only break out into a directory with multiple module files when it warrants it.